tools/HelpGeneration/Exceptions/ValidateHelpIssues.csv

```diff
@@ -305,3 +305,4 @@ Update-AzureRmIoTDeviceProvisioningServiceAccessPolicy.md,No entry was found for
 Update-AzureRmIoTDeviceProvisioningServiceLinkedHub.md,No entry was found for the online version field in the header. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/azurerm.deviceprovisioningservices/update-azurermiotdeviceprovisioningservicelinkedhub
 Get-AzureRmIoTDeviceProvisioningServiceCertificateVerificationCode.md,No entry was found for the online version field in the header. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/azurerm.deviceprovisioningservices/get-azurermiotdeviceprovisioningservicecertificateverificationcode
 New-AzureRmIoTDeviceProvisioningServiceCertificateVerificationCode.md,No entry was found for the online version field in the header. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/azurerm.deviceprovisioningservices/new-azurermiotdeviceprovisioningservicecertificateverificationcode
+Update-AzureRmIotHub.md,No entry was found for the online version field in the header. The corresponding URL should be the following: https://docs.microsoft.com/en-us/powershell/module/azurerm.iothub/update-azurermiothub
```

Comment:
Do not add this exception, instead, make the change as noted in the markdown file above

Reply:
removed